web/models.py

```
@@ -1530,6 +1530,78 @@ def __str__(self):
         return f"{self.user.username} - Current: {self.current_streak}, Longest: {self.longest_streak}"
 
 
+class SocialShareDiscount(models.Model):
+    """Model for tracking course share discounts on social media."""
+    
+    PLATFORM_CHOICES = [
+        ("twitter", "Twitter"),
+        ("facebook", "Facebook"),
+        ("linkedin", "LinkedIn"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)